src/elements/hx-div/_shadow.scss

```diff
@@ -16,7 +16,7 @@
       var(--hxPaddingBottom)
       var(--hxPaddingLeft);
 
-    #hxDiv {
+    .hx-div, #hxDiv {
```

Comment:
basically we'd have to duplicate anything that is an id or tag matcher to also offer a class variant